assets/application.js

```diff
@@ -18,7 +18,4 @@ require('govuk-template')
 require('javascripts')
 require('./components')
 require('./components/show-hide-content/show-hide-content-init')
-var GOVUK = require('stageprompt')
```

Comment:
Do we know if any consuming services are calling the functions defined in `stageprompt`? If so, should we stub them out?

Reply:
I've updated this to stub them - the functions themselves weren't previously removed but I agree that stubbing is the best option.